src/data/services/LmsApiService.js

```diff
@@ -67,6 +67,24 @@ class LmsApiService {
     const rolesUrl = `${LmsApiService.baseUrl}/api/enrollment/v1/roles/?course_id=${encodeURIComponent(courseId)}`;
     return apiClient.get(rolesUrl);
   }
+
+  static getGradeExportCsvUrl(courseId, options = {}) {
```

Comment:
nit: since the same url is used for export and import, perhaps this function name should be more genera.